.github/workflows/pull-request-management.yml

```diff
@@ -0,0 +1,16 @@
+---
+name: Code Testing
+on:
+  pull-request:
+    types: [opened, reopened]
```

Comment:
What is the rational behind limiting the scope here?

I see this in the doc:
```
Note: By default, only the opened, synchronize, and reopened activity types trigger workflows that run on the pull_request event. To trigger workflows by different activity types, use the types keyword.
```

Reply:
yes, Corrected.